package.json

```diff
@@ -1,77 +1,51 @@
 {
-  "name": "@cloudfour/eslint-plugin",
-  "version": "23.0.0",
-  "description": "Cloud Four's shareable ESLint configuration",
-  "author": "Cloud Four <info@cloudfour.com> (http://cloudfour.com)",
-  "homepage": "https://github.com/cloudfour/eslint-config",
-  "bugs": "https://github.com/cloudfour/eslint-config/issues",
-  "license": "MIT",
-  "main": "./index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/cloudfour/eslint-config.git"
-  },
-  "contributors": [
-    "Erik Jung <erik@cloudfour.com>",
-    "Caleb Eby <caleb@cloudfour.com>"
-  ],
-  "files": [
-    "dist/config.js",
-    "src/rules.js",
-    "index.js",
-    "src/rules/**/*.js",
-    "!src/rules/**/*.test.js"
-  ],
-  "engines": {
-    "node": ">=18.0.0"
-  },
-  "keywords": [
-    "eslint",
-    "eslintconfig"
-  ],
-  "dependencies": {
-    "@babel/core": "^7.23.6",
-    "@typescript-eslint/eslint-plugin": "^6.8.0",
-    "@typescript-eslint/parser": "^6.8.0",
-    "eslint-plugin-import": "^2.28.1",
-    "eslint-plugin-jsdoc": "^46.8.2",
-    "eslint-plugin-n": "^16.2.0",
-    "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-unicorn": "^48.0.1"
-  },
-  "peerDependencies": {
-    "eslint": "^8.0.0"
-  },
-  "devDependencies": {
-    "@changesets/changelog-github": "0.5.0",
-    "@changesets/cli": "2.27.1",
-    "@changesets/write": "0.3.0",
-    "eliminate": "1.1.2",
-    "eslint": "8.51.0",
-    "eslint-config-prettier": "9.0.0",
-    "eslint-config-standard": "17.1.0",
-    "eslint-config-xo": "0.43.1",
-    "eslint-formatter-pretty": "5.0.0",
-    "kleur": "4.1.5",
-    "mkdirplz": "1.0.2",
-    "powerwalker": "0.1.2",
-    "prettier": "3.0.3",
-    "prompts": "2.4.2",
-    "typescript": "5.3.3"
-  },
-  "scripts": {
-    "check-lint": "eslint --format=pretty --ext=.js,.mjs,.cjs . && prettier --check .",
-    "load-fixture-repo": "node fixtures/load-repo",
-    "lint": "eslint --format=pretty --ext=.js,.mjs,.cjs --fix . && prettier --write .",
-    "build": "node build.mjs",
-    "changeset": "changeset",
-    "version": "changeset version && prettier --write .",
-    "release": "npm run build && changeset publish"
-  },
-  "prettier": {
-    "singleQuote": true
-  },
-  "eslintConfig": {
-    "extends": "plugin:@cloudfour/recommended"
-  }
+	"name": "@cloudfour/eslint-config",
+	"version": "2.0.0-alpha.11",
+	"description": "Cloud Four's shareable ESLint configuration",
+	"author": "Cloud Four <info@cloudfour.com> (http://cloudfour.com)",
+	"homepage": "https://github.com/cloudfour/eslint-config",
+	"bugs": "https://github.com/cloudfour/eslint-config/issues",
+	"license": "MIT",
+	"main": "./eslint.config.js",
+	"type": "module",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/cloudfour/eslint-config.git"
+	},
+	"engines": {
+		"node": ">=18.0.0"
+	},
+	"keywords": [
+		"eslint",
+		"eslintconfig"
+	],
+	"peerDependencies": {
+		"eslint": ">= 9"
```

Comment:
This is the syntax ESLint recommends, which translates to "any version of ESLint 9 or greater."